src/adapters/notifications/NotificationCenterAdapter.ts

```diff
@@ -867,7 +869,7 @@ export class NotificationCenterAdapter implements NotificationAdapterInterface {
   }
 }
 
-const getSegmentDonationAttributes = (params: {
+const getSegmentDonationAttributes = async (params: {
   user: User;
```

Comment:
Please remove any redundant fields that we don't use it in notification-center

Reply:
Done!